### ISSUE TITLE
[BUG]:  download bad dub 

### ISSUE THREAD

**Issue by rcyclope** [anidl/multi-downloader-nx#631]
### Program version

v4.6.1

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

Language Options -> Dub Languages : jpn

### Show ID

GYZJ43JMR

### Episode

1, 2, .....

### Console Output

```Shell
since v4.6.1 GUI, when i try to download jpn dub language i have french dub
```

### Additional Information

![Capture d’écran 2024-04-03 220949](https://github.com/anidl/multi-downloader-nx/assets/17126922/bfb11f62-4def-42ea-860f-838eefb8e34d)

**Comment by Jaynator495** [comment]
This is probably a bug in the episode selection code for the dub with the new drm free API, does it work if you put `kstream: 2` in your cli-defaults.yml?

**Comment by AnimeDL** [comment]
Pushed commit 3ca4000ed13d11f6d4c0c5351eeb5e794474c5af which should fix the issue, release coming soon, looking to fix a couple other bugs before I push it

**Comment by AnimeDL** [comment]
Released version 4.6.2 which should fix this issue, feel free to re-open if it's still a problem

**Comment by rcyclope** [comment]
that work fine but when i multi download the programme shutdown after 1 download
